resources/assets/js/app.ts

```diff
@@ -31,7 +30,7 @@ import SettingPage from './views/setting/SettingPage.vue';
  * vue component for password reset
  */
 import ResetPage from './views/reset/ResetPage.vue';
-import PasswordRecovery from './views/reset/PasswordRecovery.vue';
+import ResetPasswordRecovery from './views/reset/ResetPasswordRecovery.vue';
```

Comment:
Why `PasswordRecovery` renamed to `ResetPasswordRecovery`?

Revert it. `/reset/PasswordRecovery` is fine